flask_github.py

```diff
@@ -24,6 +24,45 @@
 null_handler = logging.NullHandler()
 _logger.addHandler(null_handler)
 
+#: MIME type of JSON
+JSON_MIMETYPE = 'application/json'
+
+#: Generic (default) MIME type if response headers are not set
+GENERIC_MIMETYPE = 'application/octet-stream'
+
+
+def is_valid_response(response):
+    """Returns ``True`` if response ``status_code`` is not an error type,
+    raises :class:`GitHubError` otherwise.
+
+    :param response: :class:~`requests.Response` object to check
+    :type response: :class:~`requests.Response`
+    :returns: ``True`` if response ``status_code`` is not an error type,
+              ``False`` otherwise.
+    :rtype bool:
+    """
+    if response.status_code < 400:
```

Comment:
So informational and redirection status codes should be considered errors, also? I was concerned that if for some reason a redirection response as sent by the GitHub API that it would incorrectly throw, though I know previously the check was only if it was in the 200 block.